06.bowling_object/game.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative './frame'
+
+class Game
+  def initialize(scores)
+    @frames = generate_frames(scores)
+  end
+
+  def calculate_point
+    @frames.sum { |frame| frame.calculate_point(@frames) }
+  end
+
+  private
+
+  def generate_frames(scores)
+    duplicated_scores = scores.dup
+    (1..Frame::TOTAL_FRAMES).map do |index|
```

Comment:
Frameクラスのinitializeメソッドにあわせｔ

```suggestion
    (1..Frame::TOTAL_FRAMES).map do |frame_number|
```

にするとか